relinker/src/main/java/com/getkeepsafe/relinker/TextUtils.java

```diff
@@ -20,6 +20,9 @@
  * avoid depending on android.text.TextUtils.
  */
 final class TextUtils {
+    private TextUtils() {
```

Comment:
this is to prevent creation of instances of this class since this is a utils class